plotly/tools.py

```diff
@@ -251,7 +251,7 @@ def reset_config_file():
 
 ### embed tools ###
 
-def get_embed(file_owner_or_url, file_id=None, width="100%", height=525):
+def get_embed(file_owner_or_url, file_id=None, share_key=None, width="100%", height=525):
```

Comment:
Where are they going to get the fid or url in the first place though? either the web browser (which should include `share_key` in the url anyway) or the return of `py.plot`, which should also include the `share_key` in it. i vote `file_owner_or_url` detecting the presence `share_key` and not worrying about folks using the `share_key` keyword argument